rest_framework/compat.py

```diff
@@ -254,7 +249,7 @@ def md_filter_add_syntax_highlight(md):
         return False
 
 try:
-    import pytz
+    import pytz  # noqa
```

Comment:
Starting with [Django 1.11](https://docs.djangoproject.com/en/dev/releases/1.11/#pytz-is-a-required-dependency-and-support-for-settings-time-zone-none-is-removed), pytz is a Django requirement. So django-rest-framework can expect it to exist. I think adding a comment with this information would be helpful so that when Django 1.10 is dropped it is noticed and this workaround is removed.